deploy/dev-clowdenv.yaml

```diff
@@ -26,6 +26,10 @@ objects:
         port: 8000
         privatePort: 8080
         mode: operator
+        # enable envoy sidecar to handle https connections
+        tls:
+          enabled: true
+          port: 8800
```

Comment:
I don't know where this file is used, but perhaps it's good to keep this change for clarity.